package.json

```diff
@@ -16,7 +16,7 @@
   },
   "dependencies": {
     "@dylan/coerce": "~0.2.0",
-    "@dylan/engines": "github:shanebo/engines#0.1.0",
+    "@dylan/engines": "^0.1.1",
```

Comment:
@joeosburn shouldn't this be a `~`?

Reply:
Yep, the npm install changed it and I missed it. It's fixed now.